### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

**Comment by othercorey** [comment]
phpunit 7 doesn't seem to work with pcov so disabled code coverage.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+name: CakePHP CI
```
Comment (ADmad) [inline .github/workflows/ci.yml:1]: Please keep this as `CI` for consistency across all repos. The workflow name is used to URL to filter the results on Actions page, which are needed for "build status" badges.

Comment (othercorey) [reply .github/workflows/ci.yml:1]: I just copied the .github folder from another repo. Someone will have to follow up and check them all I think.

Comment (ADmad) [reply .github/workflows/ci.yml]: I'll do that whenever I get some time.

**Pull request merged**